NEWS.md

```diff
@@ -14,7 +14,8 @@
 * Identify non-riskmetric cards (#573)
 
 ### Squashed Bugs
-* Fixed busted button introduced with #547 (#592)
+* Fixed busted button introduced with #547 
+* Fixed tests showing "Package Downloads" community usage card as non-metric
```

Comment:
You'll have to remove this from the v2.0.0 release. It should go up above, near the **development version** header.